datacats/environment.py

```diff
@@ -568,6 +569,7 @@ def _run_web_container(self, port, command):
             ro = {self.datadir + '/venv': '/usr/lib/ckan'}
             volumes_from = None
 
+        print 'Binding on host {}'.format(host)
```

Comment:
Don't print from environment.py, do that in the cli module instead. And here only do it if the user specifies a non-default value